manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml

```diff
@@ -9,10 +9,10 @@ metadata:
     certified: "false"
     repository: https://github.com/maistra/istio-operator
     description: |-
-      The Service Mesh Operator provides means for installing, configuring, and managing an instance of Red Hat OpenShift Service Mesh.
+      The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh.
```

Comment:
Should this be labeled as Maistra rather than OpenShift Service Mesh?

Reply:
I see that Jonh asked this already. I'm good with it for now. I'm guessing that eventually we will want to differentiate the two though.